cmd/fmt_test.go

```diff
@@ -269,3 +290,61 @@ func TestFmtFailFileChangesDiff(t *testing.T) {
 		}
 	})
 }
+
+func TestFmtSingleWrongArityError(t *testing.T) {
+	params := fmtCommandParams{}
+	var stdout bytes.Buffer
+
+	files := map[string]string{
+		"policy.rego": singleWrongArity,
+	}
+
+	test.WithTempFS(files, func(path string) {
+		policyFile := filepath.Join(path, "policy.rego")
+		info, err := os.Stat(policyFile)
+		err = formatFile(&params, &stdout, policyFile, info, err)
+		if err == nil {
+			t.Fatalf("Expected error but did not receive one")
+		}
+
+		expectedSingleWrongArityErr := "failed to parse Rego source file: " + policyFile + ": 1 error occurred: " + policyFile + ":6: rego_type_error: plus: arity mismatch\n\thave: (var,var,var)\n\twant: (number,number) (2)"
+
+		if err.Error() != expectedSingleWrongArityErr {
+			t.Fatalf("Expected:%s\n\nGot:%s\n\n", expectedSingleWrongArityErr, err)
+		}
+	})
+}
+
+func TestFmtMultipleWrongArityError(t *testing.T) {
+	params := fmtCommandParams{}
+	var stdout bytes.Buffer
+
+	files := map[string]string{
+		"policy.rego": MultipleWrongArity,
+	}
+
+	test.WithTempFS(files, func(path string) {
+		policyFile := filepath.Join(path, "policy.rego")
+		info, err := os.Stat(policyFile)
+		err = formatFile(&params, &stdout, policyFile, info, err)
+		if err == nil {
+			t.Fatalf("Expected error but did not receive one")
+		}
+		expectedErrLines := []string{"failed to parse Rego source file: " + policyFile + ": 3 errors occurred:",
+			policyFile + ":7: rego_type_error: plus: arity mismatch",
+			"\thave: (array)",
+			"\twant: (number,number)",
+			policyFile + ":8: rego_type_error: and: arity mismatch",
+			"\thave: (boolean,boolean,boolean)",
+			"\twant: (set[any],set[any])",
+			policyFile + ":9: rego_type_error: plus: arity mismatch",
+			"\thave: (var,var,var,var)",
+			"\twant: (number,number) (2)",
+		}
+		expectedMultipleWrongArityErr := strings.Join(expectedErrLines, "\n")
```

Comment:
[style] I'd probably have gone with something like
```go
expectedMultipleWrongArityErr := fmt.Sprintf(`failed to parse Rego source file: %[1]s: 3 errors occurred:
   %[1]s:7: rego_type_error: plus: arity mismatch
...`, policyFile)
```
but that's just a personal preference. This is fine.

Reply:
The previous change should address this one as well